tests/test_site/settings.py

```diff
@@ -74,8 +74,14 @@
 
 # OpenTracing settings
 
-OPENTRACING_TRACE_ALL = True
-OPENTRACING_TRACER = django_opentracing.DjangoTracer(opentracing.Tracer())
-OPENTRACING_TRACED_ATTRIBUTES = ['META', 'FAKE_ATTRIBUTE']
+import appdash
```

Comment:
Oops. Sorry. I should not issue a PR on this. Just wanted to share the implementation but got not a very good idea on how.